### ISSUE TITLE
DbDriverClassName Can't change

### ISSUE THREAD

**Issue by MagakiReimu** [alibaba/nacos#1572]
## Issue Description
Now the program db-driver-class-name is not a environment variable。I need change the db-driver-class and db-url to establish a connection to the database mysql through a network device。

Type: *bug report* or *feature request*

### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3. 

### Tell us your environment
nacos-server 1.0.1

**Comment by nkorange** [comment]
You want to change the drive class and db url at runtime?

**Comment by MagakiReimu** [comment]
> You want to change the drive class and db url at runtime?

No need to modify it at runtime, modify it before starting

**Comment by nkorange** [comment]
> > You want to change the drive class and db url at runtime?
> 
> No need to modify it at runtime, modify it before starting

Why you need this feature?

**Comment by MagakiReimu** [comment]
> > > You want to change the drive class and db url at runtime?
> > 
> > 
> > No need to modify it at runtime, modify it before starting
> 
> Why you need this feature?

We only allow the database to be placed in the private network. The external network connection database needs to modify the database link address and database link driver. Our current micro service needs colleagues to be placed in the private network and the external network, so I need to arrange both the private network and the external network. Set nacos, private network nacos connection database no problem, the external network nacos connection database exists this problem. I think this problem should appear in some enterprises that set up private networks.

**Comment by KomachiSion** [comment]
Can you try to use latest version and retry? The issue is too old to search. If latest version still has problem, please open a new issue let us know.
